src/main/java/net/smartcosmos/extension/tenant/UserManagementPersistenceConfig.java

```diff
@@ -21,17 +15,6 @@
 @ComponentScan
 @Configuration
 @Import({ UserDetailsPersistenceConfig.class })
+public class UserManagementPersistenceConfig {
 
-public class UserManagementPersistenceConfig extends WebMvcConfigurerAdapter {
```

Comment:
Don't we need this (or the `WebMvcAutoConfigurerAdapter`) to make the converters work?